mm/shm/shm_initialize.c

```diff
@@ -69,20 +69,20 @@ struct shm_info_s g_shminfo =
 
 int shm_group_initialize(FAR struct task_group_s *group)
 {
-  DEBUGASSERT(group && !group->tg_shm.gs_handle);
+  int ret = -ENOMEM;
 
-  group->tg_shm.gs_handle =
+  GRAN_HANDLE shm_allocator =
```

Comment:
I think this is also a bit clunky, having a global SHM_VBASE and SHM_MAXPAGES should be unnecessary, also for mapping files and ftruncate this is unusable. But fixing this is another PR.

Reply:
These are part of SYS-V inspired shm, and are not really related to this PR, which only try to add task group specific list of  VM mappings, in order to be able to do munmap.